src/helpers/purchase-result-helpers.ts

```diff
@@ -0,0 +1,25 @@
+import type { CustomerInfo } from "../entities/customer-info";
+import type { Package } from "../entities/offerings";
+
+export function getRenewalDateFromPackage(
```

Comment:
Hmm I think this could directly live in the WPL, unless we think this is something other users of the SDK will need... Which they can still access through the CustomerInfo as I mentioned.